.gitignore

```diff
@@ -1,60 +1,65 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
@apolkingg8 It's difficult to review a file when line endings change. Is this something that will happen often?